README.md

```diff
@@ -33,60 +33,9 @@ Indefinitely harvests an RPDE feed, following the ["expected consumer behaviour"
 
 **N.B.** This function will run indefinitely, and only return if a fatal error occurs. For this reason, you will generally **not** want to run `await harvestRPDE(..)`.
 
-#### Required Parameters
```

Comment:
It is a hassle to keep these up-to-date alongside the type definitions and JSDocs. These should either not exist or be auto-generated.